scripts/clients/resend.ts

```diff
@@ -0,0 +1,36 @@
+import type { UserConfig } from "@hey-api/openapi-ts";
+
+export default {
+  input:
+    "https://raw.githubusercontent.com/resend/resend-openapi/main/resend.yaml",
+  output: {
+    path: "alchemy/src/resend/api",
+    format: "biome",
+  },
+  plugins: [
+    {
+      name: "@hey-api/typescript",
+      exportFromIndex: false,
+    },
+    {
+      name: "@hey-api/sdk",
+      instance: "ResendClient",
+      exportFromIndex: false,
+      auth: false,
+    },
+    {
+      name: "@hey-api/client-fetch",
+      throwOnError: true,
+    },
+  ],
+} satisfies UserConfig;
+
+export const patch = async () => {
+  const file = Bun.file("alchemy/src/resend/api/sdk.gen.ts");
+  let text = await file.text();
+  text = text.replace(
+    "import",
+    'import type {ResendError} from "../api.ts"\nimport',
+  );
+  await file.write(text.replaceAll(/unknown,/g, "ResendError,"));
+};
```

Comment:
Their OpenAPI spec doesn't have any errors, so this is a hack to sub in our own error type.